### DISCUSSION TITLE
Issue with Customer class namespace duplicate

### DISCUSSION THREAD

**Discussion by TS1997** [pimcore/pimcore#10177] [Q&A / Support]
I'm trying to create login functionality on a site using the Pimcore CMF framework (Pimcore 10.1.1 & CMF 3.2.1) but have an issue where there's a Customer class duplicate that causes issues as the wrong one is loaded. The one I'm after exists in `var/classes/DataObject` as it should but I also have one under `vendor/pimcore/pimcore/.github/ci/files/var/classes/DataObject` which is loaded instead and is therefore causing issues. Why is that one there? I tried deleting the vendor folder and performing a clean composer install but it came back. Is this an issue on your end or is there something I can do to fix this?

**Comment by jdreesen** [comment]
Those files are CI test files, you don't need these. 

I created [a PR](https://github.com/pimcore/pimcore/pull/10181) that excludes these files from the package downloaded by Composer (when required as `dist`).

But in fact, I'm not sure why the autoloader picks it up at all.

**Reply by TS1997** [reply]
I thought that was weird as well but lets hope they accept your PR soon so I can continue without having to manually modify the vendor folder after each update etc
